concreate/cli.py

```diff
@@ -45,10 +47,21 @@ def run(self):
         build_group = parser.add_argument_group(
             'build', "Arguments valid for the 'build' target")
 
+        build_group.add_argument('--build-engine',
+                                 default='docker',
+                                 choices=['docker', 'osbs'],
+                                 help='An engine used to build an image [docker,osbs]')
+
         build_group.add_argument('--build-tag',
                                  dest='build_tags',
                                  action='append',
-                                 help='tag to assign to the built image, can be used multiple times')
+                                 help='Tag to assign to the built image, can be used multiple times')
+
+        build_group.add_argument('--build-osbs-release',
+                                 dest='build_osbs_release',
+                                 action='store_true',
+                                 default=False,
+                                 help='Do a proper OSBS build not scratch only.')
```

Comment:
I would probably reword this and drop "scratch". Hard to define what "proper" is :) "Execute release build". We can and maybe should even highlight that this is important switch.